packages/framework/src/schemas/steps/actions/index.ts

```diff
@@ -1,5 +1,6 @@
 import { ActionStepEnum } from '../../../constants';
 import { Schema } from '../../../types/schema.types';
+
 import { delayActionSchemas } from './delay.schema';
```

Comment:
We can revert this whitespace change.

```suggestion
import { Schema } from '../../../types/schema.types';
import { delayActionSchemas } from './delay.schema';
```